2023-03-07 14:05:55,383 INFO    ======================================================================
2023-03-07 14:05:55,384 INFO    Survey 160, for Ouro Preto, Minas Gerais
2023-03-07 14:05:55,385 INFO    Searching by bounding box, max_zoom=24
2023-03-07 14:05:55,385 INFO    ----------------------------------------------------------------------
2023-03-07 14:05:55,386 INFO    Rectangle calculated: [-20.2519, -43.4372, -20.5229, -43.7846]
2023-03-07 14:05:55,386 INFO    Searching rectangle: zoom factor = 0, node = []
2023-03-07 14:05:56,201 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:06:03,450 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:06:13,025 INFO    Page 01 returned 18 listings
2023-03-07 14:06:14,766 INFO    Room 45452801: insert failed
2023-03-07 14:06:14,769 INFO    Room 47939876: insert failed
2023-03-07 14:06:14,771 INFO    Room 770977262407924429: insert failed
2023-03-07 14:06:14,774 INFO    Room 568273495067211791: insert failed
2023-03-07 14:06:14,777 INFO    Room 30403033: insert failed
2023-03-07 14:06:14,779 INFO    Room 811376203986675036: insert failed
2023-03-07 14:06:14,782 INFO    Room 787734071865852313: insert failed
2023-03-07 14:06:14,785 INFO    Room 41078755: insert failed
2023-03-07 14:06:14,787 INFO    Room 650587689817510894: insert failed
2023-03-07 14:06:14,789 INFO    Room 758263738755481654: insert failed
2023-03-07 14:06:14,792 INFO    Room 16844691: insert failed
2023-03-07 14:06:14,795 INFO    Room 731014935944515463: insert failed
2023-03-07 14:06:14,798 INFO    Room 732426078608851583: insert failed
2023-03-07 14:06:14,800 INFO    Room 52305704: insert failed
2023-03-07 14:06:14,802 INFO    Room 841172626179134958: insert failed
2023-03-07 14:06:14,805 INFO    Room 833306856028758728: insert failed
2023-03-07 14:06:14,807 INFO    Room 839664200962255692: insert failed
2023-03-07 14:06:14,810 INFO    Room 53346940: insert failed
2023-03-07 14:06:14,811 INFO    Page 02 returned 18 listings
2023-03-07 14:06:17,361 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:06:25,707 INFO    Page 03 returned 18 listings
2023-03-07 14:06:27,039 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:06:35,071 INFO    Page 04 returned 18 listings
2023-03-07 14:06:37,791 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:06:47,497 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:06:53,717 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:07:02,668 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:07:17,422 WARNING HTTP status 403 from web site: IP address blocked. Waiting 0.1 minutes.
2023-03-07 14:07:26,790 INFO    Page 05 returned 18 listings
2023-03-07 14:07:32,236 INFO    Page 06 returned 18 listings
2023-03-07 14:07:34,697 INFO    Page 07 returned 18 listings
2023-03-07 14:07:40,734 INFO    Page 08 returned 18 listings
2023-03-07 14:07:49,881 INFO    Page 09 returned 00 listings
2023-03-07 14:07:49,889 INFO    Final page of listings for this search
2023-03-07 14:07:49,890 INFO    Results: 9 pages, 126 new rooms
2023-03-07 14:07:49,894 INFO    Finishing survey 160, for Ouro Preto, Minas Gerais
2023-03-07 14:07:49,907 INFO    Initialing search by overall satisfactions
2023-03-07 14:09:06,903 INFO    Searching for comodities
2023-03-07 14:10:08,348 INFO    Searching for comodities
2023-03-07 14:10:44,534 INFO    Searching for comodities
2023-03-07 14:11:23,601 INFO    Searching for comodities
2023-03-07 14:11:59,090 INFO    Searching for comodities
2023-03-07 14:12:53,234 INFO    Searching for comodities
2023-03-07 14:13:33,387 INFO    Searching for comodities
2023-03-07 14:14:09,542 INFO    Searching for comodities
2023-03-07 14:14:44,926 INFO    Searching for comodities
2023-03-07 14:15:19,866 INFO    Searching for comodities
2023-03-07 14:15:57,478 INFO    Searching for comodities
2023-03-07 14:16:53,555 INFO    Searching for comodities
2023-03-07 14:17:33,401 INFO    Searching for comodities
2023-03-07 14:18:09,618 INFO    Searching for comodities
2023-03-07 14:18:45,629 INFO    Searching for comodities
2023-03-07 14:19:21,156 INFO    Searching for comodities
2023-03-07 14:19:55,448 INFO    Searching for comodities
2023-03-07 14:20:50,859 INFO    Searching for comodities
2023-03-07 14:21:31,623 INFO    Searching for comodities
2023-03-07 14:22:07,756 INFO    Searching for comodities
2023-03-07 14:22:43,766 INFO    Searching for comodities
2023-03-07 14:23:19,029 INFO    Searching for comodities
2023-03-07 14:23:56,517 INFO    Searching for comodities
2023-03-07 14:24:56,608 INFO    Searching for comodities
2023-03-07 14:25:35,736 INFO    Searching for comodities
2023-03-07 14:26:10,228 INFO    Searching for comodities
2023-03-07 14:26:45,413 INFO    Searching for comodities
2023-03-07 14:27:21,127 INFO    Searching for comodities
2023-03-07 14:27:56,661 INFO    Searching for comodities
2023-03-07 14:28:52,365 INFO    Searching for comodities
2023-03-07 14:29:31,194 INFO    Searching for comodities
2023-03-07 14:30:06,753 INFO    Searching for comodities
2023-03-07 14:30:41,529 INFO    Searching for comodities
2023-03-07 14:31:17,444 INFO    Searching for comodities
2023-03-07 14:31:54,305 INFO    Searching for comodities
2023-03-07 14:32:51,154 INFO    Searching for comodities
2023-03-07 14:33:32,868 INFO    Searching for comodities
2023-03-07 14:34:07,656 INFO    Searching for comodities
2023-03-07 14:34:42,580 INFO    Searching for comodities
2023-03-07 14:35:17,468 INFO    Searching for comodities
2023-03-07 14:35:52,683 INFO    Searching for comodities
2023-03-07 14:36:51,716 INFO    Searching for comodities
2023-03-07 14:37:33,831 INFO    Searching for comodities
2023-03-07 14:38:10,017 INFO    Searching for comodities
2023-03-07 14:38:45,274 INFO    Searching for comodities
2023-03-07 14:39:20,714 INFO    Searching for comodities
2023-03-07 14:39:55,319 INFO    Searching for comodities
2023-03-07 14:40:50,873 INFO    Searching for comodities
2023-03-07 14:41:30,414 INFO    Searching for comodities
2023-03-07 14:42:05,828 INFO    Searching for comodities
2023-03-07 14:42:43,464 INFO    Searching for comodities
2023-03-07 14:43:19,349 INFO    Searching for comodities
2023-03-07 14:43:55,740 INFO    Searching for comodities
2023-03-07 14:44:52,888 INFO    Searching for comodities
2023-03-07 14:45:31,217 INFO    Searching for comodities
2023-03-07 14:46:05,533 INFO    Searching for comodities
2023-03-07 14:46:40,650 INFO    Searching for comodities
2023-03-07 14:47:17,034 INFO    Searching for comodities
2023-03-07 14:47:51,726 INFO    Searching for comodities
2023-03-07 14:48:47,053 INFO    Searching for comodities
2023-03-07 14:49:27,073 INFO    Searching for comodities
2023-03-07 14:50:02,001 INFO    Searching for comodities
2023-03-07 14:50:37,420 INFO    Searching for comodities
2023-03-07 14:51:13,020 INFO    Searching for comodities
2023-03-07 14:51:56,044 INFO    Searching for comodities
2023-03-07 14:52:53,384 INFO    Searching for comodities
2023-03-07 14:53:34,043 INFO    Searching for comodities
2023-03-07 14:54:13,381 INFO    Searching for comodities
2023-03-07 14:54:49,687 INFO    Searching for comodities
2023-03-07 14:55:27,174 INFO    Searching for comodities
2023-03-07 14:56:04,069 INFO    Searching for comodities
2023-03-07 14:57:00,512 INFO    Searching for comodities
2023-03-07 14:57:42,897 INFO    Searching for comodities
2023-03-07 14:58:19,320 INFO    Searching for comodities
2023-03-07 14:58:56,240 INFO    Searching for comodities
2023-03-07 14:59:32,938 INFO    Searching for comodities
2023-03-07 15:00:09,813 INFO    Searching for comodities
2023-03-07 15:01:12,725 INFO    Searching for comodities
2023-03-07 15:01:53,740 INFO    Searching for comodities
2023-03-07 15:02:30,013 INFO    Searching for comodities
2023-03-07 15:03:05,121 INFO    Searching for comodities
2023-03-07 15:03:38,796 INFO    Searching for comodities
2023-03-07 15:04:16,752 INFO    Searching for comodities
2023-03-07 15:05:18,776 INFO    Searching for comodities
2023-03-07 15:06:05,064 INFO    Searching for comodities
2023-03-07 15:06:46,293 INFO    Searching for comodities
2023-03-07 15:07:28,461 INFO    Searching for comodities
2023-03-07 15:08:15,151 INFO    Searching for comodities
2023-03-07 15:08:53,513 INFO    Searching for comodities
2023-03-07 15:09:51,155 INFO    Searching for comodities
2023-03-07 15:10:32,276 INFO    Searching for comodities
2023-03-07 15:11:09,586 INFO    Searching for comodities
2023-03-07 15:11:46,591 INFO    Searching for comodities
2023-03-07 15:12:24,118 INFO    Searching for comodities
2023-03-07 15:12:59,656 INFO    Searching for comodities
2023-03-07 15:13:58,805 INFO    Searching for comodities
2023-03-07 15:14:38,615 INFO    Searching for comodities
2023-03-07 15:15:14,395 INFO    Searching for comodities
2023-03-07 15:15:49,659 INFO    Searching for comodities
2023-03-07 15:16:40,151 INFO    Searching for comodities
2023-03-07 15:17:18,268 INFO    Searching for comodities
2023-03-07 15:18:22,830 INFO    Searching for comodities
2023-03-07 15:19:10,960 INFO    Searching for comodities
2023-03-07 15:19:50,905 INFO    Searching for comodities
2023-03-07 15:20:31,913 INFO    Searching for comodities
2023-03-07 15:21:11,043 INFO    Searching for comodities
2023-03-07 15:21:47,721 INFO    Searching for comodities
2023-03-07 15:22:44,771 INFO    Searching for comodities
2023-03-07 15:23:26,477 INFO    Searching for comodities
2023-03-07 15:24:03,886 INFO    Searching for comodities
2023-03-07 15:24:41,103 INFO    Searching for comodities
2023-03-07 15:25:18,501 INFO    Searching for comodities
2023-03-07 15:25:56,145 INFO    Searching for comodities
2023-03-07 15:26:54,326 INFO    Searching for comodities
2023-03-07 15:27:36,277 INFO    Searching for comodities
2023-03-07 15:28:12,753 INFO    Searching for comodities
2023-03-07 15:28:50,092 INFO    Searching for comodities
2023-03-07 15:29:27,308 INFO    Searching for comodities
2023-03-07 15:30:06,658 INFO    Searching for comodities
2023-03-08 08:58:56,168 INFO    Searching for comodities
2023-03-08 08:59:45,095 ERROR   WebDriverException
2023-03-08 09:00:03,677 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000020D37B4ECD0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /reverse?lat=-20.38531&lon=-43.50038&format=json&accept-language=es&addressdetails=1
2023-03-08 09:00:03,766 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000020D37B4EF50>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /reverse?lat=-20.38531&lon=-43.50038&format=json&accept-language=es&addressdetails=1
2023-03-08 09:00:22,232 ERROR   Top level exception handler: quitting.
Traceback (most recent call last):
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\util\connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\socket.py", line 962, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connection.py", line 358, in connect
    self.sock = conn = self._new_conn()
                       ^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x0000020D37754590>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\requests\adapters.py", line 489, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connectionpool.py", line 815, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connectionpool.py", line 815, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\connectionpool.py", line 787, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Max retries exceeded with url: /reverse?lat=-20.38531&lon=-43.50038&format=json&accept-language=es&addressdetails=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000020D37754590>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\geopy\adapters.py", line 457, in _request
    resp = self.session.get(url, timeout=timeout, headers=headers)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\requests\sessions.py", line 600, in get
    return self.request("GET", url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\requests\adapters.py", line 565, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Max retries exceeded with url: /reverse?lat=-20.38531&lon=-43.50038&format=json&accept-language=es&addressdetails=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000020D37754590>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\mono\airbnb-data-collection\airbnb.py", line 847, in main
    full_process(ab_config, args.full_survey)
  File "C:\mono\airbnb-data-collection\airbnb.py", line 797, in full_process
    update_routes_geolocation(ab_config, search_area_name)
  File "C:\mono\airbnb-data-collection\airbnb.py", line 263, in update_routes_geolocation
    location.reverse_geocode(config)
  File "C:\mono\airbnb-data-collection\airbnb_geocoding.py", line 146, in reverse_geocode
    address = reverse(self.lat_round + ", " + self.lng_round)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\geopy\geocoders\nominatim.py", line 372, in reverse
    return self._call_geocoder(url, callback, timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\geopy\geocoders\base.py", line 368, in _call_geocoder
    result = self.adapter.get_json(url, timeout=timeout, headers=req_headers)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\geopy\adapters.py", line 447, in get_json
    resp = self._request(url, timeout=timeout, headers=headers)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Usuario\AppData\Local\Programs\Python\Python311\Lib\site-packages\geopy\adapters.py", line 469, in _request
    raise GeocoderUnavailable(message)
geopy.exc.GeocoderUnavailable: HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Max retries exceeded with url: /reverse?lat=-20.38531&lon=-43.50038&format=json&accept-language=es&addressdetails=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000020D37754590>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
